syntax = "proto3";

package ProtoTransport;
import "db.proto";
import "router.proto";
import "renderer.proto";

message TransportCatalog {
    Db db = 1;
    Router router = 2;
    Renderer renderer = 3;
}